continuous-delivery/update-version.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+import os
+import re
+import subprocess
+
+tag = subprocess.check_output(['git', 'describe', '--tags'])
+# strip the leading v
+version = str(tag[1:].strip(), 'utf8')
+setup_path = os.path.join(os.path.dirname(__file__), '..', 'setup.py')
+print("Updating setup.py to version {}".format(version))
+with open(setup_path, 'r+') as setup_py:
+    contents = setup_py.read(64 * 1024)
```

Comment:
It's a bit scary to pass in a "this should always be big enough" value

I think read() with no arguments will read the whole file
or maybe there's a readall() function?
I have a hard time determining the exact API with the ambiguous [io](https://docs.python.org/3/library/io.html) API.